src/components/note/__internal__/note.component.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Editor, EditorState } from 'draft-js';
+import tagComponent from '../../../utils/helpers/tags/tags';
+import {
+  StyledNote, NoteContent, InlineControl, Title, Footer, FooterContent
+} from './note.style.js';
+import StatusWithTooltip from './status-with-tooltip';
+
+const Note = (props) => {
+  const {
+    noteContent,
+    width,
+    inlineControl,
+    title,
+    name,
+    createdDate,
+    status,
+    ...rest
+  } = props;
+
+  const [showTooltip, setShowTooltip] = useState(false);
+
+  const renderStatus = () => {
+    if (!status || !status.text || !status.text.length) {
+      return null;
+    }
+
+    const { text, timeStamp } = status;
+
+    return (
+      <FooterContent>
+        <StatusWithTooltip { ...tooltipProps(timeStamp, showTooltip, setShowTooltip) }>
```

Comment:
will we have here a `data-component` tag for cypress tests?

Reply:
Yeah good point I missed that